sandbox/libs/authn/src/main/java/org/opensearch/authn/PermissionStore.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.authn;
+
+import java.util.ArrayList;
```

Comment:
We shouldn't use a specific collections implementation, lets use the more generic `List`.

Reply:
No problem.